Sources/Apodini/Semantic Model Builder/gRPC/GRPCMessage.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  GRPCMessage.swift
+//  
+//
+//  Created by Moritz Schüll on 07.01.21.
+//
+
+import Foundation
+
+class GRPCMessage: Apodini.ExporterRequest {
+    /// Default message  that can be used to call handlers in cases
+    /// where no input message was provided.
+    /// Content is empty, length is zero, and the compressed flag is not set.
+    static let DefaultMessage = GRPCMessage(from: Data(), length: 0, compressed: false)
+
+    internal var data: Data
+    var length: Int
+    var compressed: Bool
+
+    init(from data: Data, length: Int, compressed: Bool) {
+        self.data = data
+        self.length = length
+        self.compressed = compressed
+    }
+
+    /// TRUE if all fragments for this message have been collected.
+    /// FALSE othwise.
+    var isComplete: Bool {
+        !(data.count < length)
```

Comment:
```suggestion
        data.count >= length
```

And I'd remove the comment or find a more specific name, e.g., `didCollectAllFragments`